api/types/device.go

```diff
@@ -233,24 +227,20 @@ func ResourceEnrollStatusToString(enrollStatus devicepb.DeviceEnrollStatus) stri
 		return "enrolled"
 	case devicepb.DeviceEnrollStatus_DEVICE_ENROLL_STATUS_NOT_ENROLLED:
 		return "not_enrolled"
-	case devicepb.DeviceEnrollStatus_DEVICE_ENROLL_STATUS_UNSPECIFIED:
-		return "unspecified"
 	default:
-		return enrollStatus.String()
```

Comment:
This is mixing truly unknown proto values with "unspecified", a bad idea in terms of resistance to future changes. The changed code is not equivalent to what it was before.